### ISSUE TITLE
Feature request: navigation of lists

### ISSUE THREAD

**Issue by exelanz** [oracle/oci-cli#636]
Hi, 
Interactive mode is fantastic. I have one request to make it even better.

If I am presented a very long list - for example all the user or all the compartments - it would be nice if I could type the first letters and a filter based on those characters is automatically applied.

This makes navigating large list much more easy and fast.

Thanks!

**Comment by karthik-k-kamath** [comment]
Thanks @exelanz for the suggestion.
 cc @KartikShrikantHegde

**Comment by kevco-us** [comment]
Doesn't it already do this?   It works for me... 
Type a few letters of the resource name and wait a few seconds for the short list to pop up.  
The filter searches anywhere in the name - not just the start of the name.

**Comment by exelanz** [comment]
Hi, yes indeed. The issue was that it takes some while before the filtered list appears. I am probably not the only one to come across this hurdle. But glad it works :-)

**Comment by kevco-us** [comment]
I agree that it could work faster.  It appears to be sending a a request to get (another) list of resources, then filtering the results.  But it should already have the full list of resources, so there's no need for a delay.
